frontend/test/integration/TestEntityProvider.ts

```diff
@@ -234,16 +257,16 @@ export function getBaseStudent(id: string): IStudent {
         status: Status.maybe,
         additionalStudentInfo: "He likes it like that",
         studies: ["I love to Spring Spring in java Spring!"],
-        skills: ["Gaming on a nice chair", "programming whilst thinking about sleeping"],
+        skills: ["other"],
```

Comment:
Why?

Reply:
Skills are not restricted in naming? A skill will never be called other.

Reply:
Student skills are restricted in length. The change to "other" is to be able to use the defined baseSkillType in the tests.

Reply:
Oh okey, I see. I think it's better to alter the back end field type for this.